server/src/optima/dbmodels.py

```diff
@@ -46,7 +46,9 @@ class ProjectDb(db.Model):
                                 uselist=False)
     project_data = db.relationship('ProjectDataDb', backref='projects',
                                 uselist=False)
-    creation_time = db.Column(db.DateTime(timezone=True), server_default=text('now()'))
+    creation_time = db.Column(db.DateTime(timezone=True), default=db.func.now())
```

Comment:
If you save the result then you modify the json :) I still don't think there's a difference, and "Last updated" should reflect any change in either the "model" or the "project".